README.md

```diff
@@ -1,7 +1,21 @@
 # custom-components-examples
 
+![Video of Custom Components](screenshots/customComponents.gif)
```

Comment:
The GIF is pretty low quality. I think let's replace with the main hero image from the website. It's here: 

https://assets-global.website-files.com/627d359d0e0aa25158817bf0/64ef667a238938b55c05e99e_CustomComponents_HeroImage_FINAL.png